<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="pathTextBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>441</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectRootButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectRootPath</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectSubButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectSubPath</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;说明:让SubPath下的每个文件在RootPath下都只有一份.如果两个文件名相同并且Size相同那么,程序就会删除其中不在SubPath下的那份文件.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCopyButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>151</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>DeleteCopy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
